application/src/main/java/org/opentripplanner/transit/api/request/RegularStopRequest.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.transit.api.request;
+
+import javax.annotation.Nullable;
+import org.locationtech.jts.geom.Envelope;
+import org.opentripplanner.transit.model.site.RegularStop;
+
+/**
+ * A request for {@link RegularStop}s.
+ * <p/>
+ * This request is used to retrieve {@link RegularStop}s that match the provided criteria.
+ */
+public class RegularStopRequest {
+
+  private final Envelope envelope;
```

Comment:
Consider creating a `WgsEnvelope` and use it here instead, similar to the `WgsCoordinate`. Maybe, we should do it as a separate PR.

Reply:
Yes, that seems like it belongs in a separate PR. Created a small issue to track it: https://github.com/opentripplanner/OpenTripPlanner/issues/6335